sci_strategic_plugin/src/main.cpp

```diff
@@ -50,6 +50,7 @@ int main(int argc, char** argv)
   pnh.param<std::string>("lane_following_plugin_name",       config.lane_following_plugin_name, config.lane_following_plugin_name);
   pnh.param<std::string>("intersection_transit_plugin_name", config.intersection_transit_plugin_name, config.intersection_transit_plugin_name);
   pnh.getParam("/vehicle_id", config.vehicle_id);
+  pnh.param<double>("vehicle_status_generation_frequency_", config.vehicle_status_generation_frequency_, 10.0);
```

Comment:
param value should be set in config/parameters.taml, so they can be changed during tests